guardian/editions
# DefaultInfoTextWebview: open links outside of webview

## Summary

Put back proper links into our webview, in this case the Privacy Policy. We got rid of them because they would open in the webview itself, with no navigation control. This changeset makes it so it opens externally (ex. the system browser).

## Test Plan

1. Open the Privacy Policy page
2. Click on the "privacy policy" link (the one for the website).
3. Observe: it opens in the browser
4. Repeat steps 1-3 for Android and iOS.
